filebeat/docs/multiple-prospectors.asciidoc

```diff
@@ -18,8 +23,11 @@ filebeat:
     -
       paths:
         - "/var/log/apache2/*"
+      include_lines: ["apache"]
+      exclude_lines: ["^DBG"]
```

Comment:
I would actually add fields as the first thing to mention. I think currently this is the most common reason people use multiple prospectors as the assign fields to the files